_ml-commons-plugin/index.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: default
+title: About ML Commons 
+nav_order: 1
+has_children: false
+has_toc: false
+---
+
+# ML Commons plugin
+
+ML Commons for OpenSearch eases the development of machine learning features by providing a set of common machine learning (ML) algorithms through transport and REST API calls. Those calls choose the right nodes and resources for each ML request and monitors ML tasks to ensure uptime. This allows you to leverage existing open-source ML algorithms and reduce the effort required to develop new ML features. 
+
+Models trained through the ML Commons plugin support model-based algorithms such as kmeans or Linear Regression. To get the best results, make sure you train your model first, then use the model to apply predictions. 
```

Comment:
"Models trained through the ML Commons plugin support model-based algorithms such as kmeans or Linear Regression" -> "ML Commons plugin supports several popular ML algorithms such as kmeans or Linear Regression".

"make sure you train your model first" -> "make sure you train your model well first"

Reply:
I don't understand the distinction between "train your model" vs "train your model well"?

Reply:
Sure, we have two ways to train model and get prediction result.
1. train model first with training data, we will get trained model, then we can use the trained model to predict some test data to check if the model trained **well** or not, for example user can check the model precision. If the model is well trained, then user can apply that model to predict new data safely. If the model is not well trained, user may need to delete that model and train a new model with different parameters and input data. May need several iterations to get a **well** trained model. 
2. Train&predict with the same data set via "train and predict" API. In this way, we don't store model at all. So we don't have model id. That means user have no way to test the trained model with other data set to evaluate how the model performs like if the precision good enough. The "train and predict" is more like some quick way to explore and get response quickly.

So, as this sentence is for "To get the best results",  I think user should use the first way to train the model and test to verify if model is good enough, then apply the model to predict.